cypress/support/fragments/check-out-actions/check-out-actions.js

```diff
@@ -81,5 +83,10 @@ export default {
 
   checkItem:(barcode) => {
     cy.expect(MultiColumnList({ id:'list-items-checked-out' }).find(HTML(including(barcode))).absent());
-  }
+  },
+
+  waitLoading:() => {
+    cy.expect(patronBarcodeField.exists());
+    cy.expect(itemBarcodeField.exists());
+  },
```

Comment:
we have this method waitLoading() created in the checkout fragment. Please use the method from the checkout fragment